<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductDialog</class>
 <widget class="QDialog" name="ProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>36</y>
     <width>96</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 127);</string>
   </property>
   <property name="text">
    <string>产品名字（中）：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="mWeight">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>37</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
   <property name="value">
    <double>49.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>32</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电视柜</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>42</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>包裹毛重：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="jWeight">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>64</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
   <property name="value">
    <double>48.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>69</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>包裹净重：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="volume">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>92</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
   <property name="value">
    <double>0.154000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>97</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>包裹体积：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="priceIn">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>200</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
   <property name="value">
    <double>2800.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>203</y>
     <width>192</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>产品进价（本币，并非包裹进价）：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="priceOut">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>225</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
   <property name="value">
    <double>370.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>229</y>
     <width>192</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>产品售价（外币，并非包裹售价）：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>17</y>
     <width>96</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>产品数量每包裹：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pakUnit">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>10</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="number">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>250</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>256</y>
     <width>156</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>产品数量（非包装数量）：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>131</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>出口退税：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tTax">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>126</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
   <property name="value">
    <double>15.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>6</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>156</y>
     <width>48</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>消费税：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xTax">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>151</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>155</y>
     <width>6</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_44">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>181</y>
     <width>48</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>增殖税：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="zTax">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>176</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
   <property name="value">
    <double>17.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_45">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>6</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>58</y>
     <width>96</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 127);</string>
   </property>
   <property name="text">
    <string>产品名字（英）：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEn">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>56</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TV CABINET</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>15</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 127);</string>
   </property>
   <property name="text">
    <string>商品编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="type">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>81</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>家具</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>83</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 127);</string>
   </property>
   <property name="text">
    <string>所属类别：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="haiguan">
   <property name="geometry">
    <rect>
     <x>378</x>
     <y>103</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>9403609990</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>106</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 127);</string>
   </property>
   <property name="text">
    <string>海关代码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>8</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>28004</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="carton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>174</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CARTON</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>154</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 127);</string>
   </property>
   <property name="text">
    <string>销售单位：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pc">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>152</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>177</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 127);</string>
   </property>
   <property name="text">
    <string>包装单位：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>132</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 127);</string>
   </property>
   <property name="text">
    <string>产地：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="birth">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CHINA</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descZh">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>241</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;型号：jd-253，材质：板式，开启方式：推拉式，颜色：橡木色，规格：120*60*45cm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descEn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;MODEL:JD-253,MATERIAL:PLATE,OPEN MODE:PUSH PULL TYPE,COLOR:OAK,SPECIFICATION:120*60*45CM&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string>中文描述：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0)</string>
   </property>
   <property name="text">
    <string>英文描述：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;注意：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#5500ff;&quot;&gt;(蓝色字体)&lt;/span&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;为每个产品计算时候必须填写项目&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#ff0000;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#aa007f;&quot;&gt;(暗粉色字体)&lt;/span&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;是自动导出表格所需要的数据&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProductDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProductDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
